notdiamond/llms/config.py

```diff
@@ -55,8 +55,11 @@ def __init__(
         system_prompt: Optional[str] = None,
         context_length: Optional[int] = None,
         input_price: Optional[float] = None,
```

Comment:
Why are we keeping the old parameters if you're renaming them to `custom_*`?

Reply:
Backward compatibility

Reply:
Note that they're kept, but undocumented, and the default is to check the `custom_*` variants

Reply:
Ok makes sense

Reply:
When you merge this, can you also update the docs too?